src/progressbar/progressbar.spec.ts

```diff
@@ -87,12 +87,12 @@ describe('ng-progressbar', () => {
       // this might fail in IE11 if attribute binding order is not respected for the <progress> element:
       // <progress [value]="" [max]=""> will fail with value = 1
       // <progress [max]="" [value]=""> will work with value = 10
```

Comment:
Is this comment still relevant if we are using the `value` property on the element?